sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_options.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <azure/core/http/http.hpp>
+#include <azure/core/response.hpp>
+#include <azure/keyvault/common/keyvault_pipeline.hpp>
+
+#include "azure/keyvault/keys/key_vault_key.hpp"
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
+
+  enum class ServiceVersion
+  {
+    V7_0,
+    V7_1
```

Comment:
Should also add V7_2 - it's coming very soon and some of these features here will need it.